### ISSUE TITLE
Hindi translation for "How to start using WordPress Playground"

### ISSUE THREAD

**Issue by sumitsinghwp** [WordPress/Learn#1771]
# Details
- Link to original content: https://learn.wordpress.org/tutorial/how-to-start-using-wordpress-playground/
- Link to original content GitHub issue: 
- Language you'll be translating to: Hindi
- Have you arranged for someone to review this translation?: Yes or No
- Reviewer's GitHub username: 
- Other info: 

# Next Steps
Once translated, please link or upload your translated files in a comment on this issue, and request a [translation review](https://make.wordpress.org/training/handbook/content-localization/#translation-review).

**Comment by sumitsinghwp** [comment]
I have translated the page and here is the Google Doc file link. If anyone has feedback then comment on the file.
[Google Doc file link](https://docs.google.com/document/d/1nA8G5dYkr0IW6HIgr7HsL3mk0ol-7BcbQNS2Dmd3Tog/edit?usp=sharing)

**Comment by shail-mehta** [comment]
@bsanevans @sumitsinghwp 

Can I review this Translation?

**Comment by webtechpooja** [comment]
Yes @shail-mehta, you can review this translation and provide your valuable feedback.

**Comment by shail-mehta** [comment]
Hi @sumitsinghwp 

**Need to Do 2 Small corrections Please check & update:**
- Please Use email playground@wordpress.net Instead of PlayStation@wordpress.net (as per Original Tutorial).
- Please Changed this wordpress-playground.zip Instead of WordPress-playground.zip

**Comment by sumitsinghwp** [comment]
Hey @shail-mehta ,

I have updated your corrections. Thank you very much for your feedback.

**Comment by shail-mehta** [comment]
Thank you @sumitsinghwp for Quick Corrections.

**Comment by shail-mehta** [comment]
@webtechpooja

 I have Reviewed the Translations and It's Looks good to me.

**Comment by shail-mehta** [comment]
@webtechpooja can you please do needful?

Hindi translation has been thoroughly reviewed.

**Comment by sumitsinghwp** [comment]
@webtechpooja, i have added content here https://learn.wordpress.org/?p=43207&post_type=wporg_workshop&preview_id=43207 and I am making a video in Hindi for this topic.

**Comment by webtechpooja** [comment]
Hi, @sumitsinghwp Thank you for all your contribution. As this is a tutorial, it should be added into the Tutorial section.

**Comment by rjekic** [comment]
@sumitsinghwp 

Since this tutorial is converted to a lesson, we can use the translation you created and use it for the translation of the lesson -> https://learn.wordpress.org/lesson/how-to-start-using-wordpress-playground/. But, there are slight differences, and you need to translate or update only the headings in the lesson document (there were no headings in the tutorial). Could you please recheck the lesson and update the translation doc so we can publish this lesson asap?

So, basically, we would need these titles translated into Hindi:

1. Resources
2. Learning outcomes
3. Transcript
4. Introduction
5. Installing plugins
6. Installing plugins and themes automatically
7. Building a theme
8. Building a site
9. Exporting a site
10. Compatibility testing

Thank you.
